x/campaign/types/errors.go

```diff
@@ -18,4 +18,6 @@ var (
 	ErrAccountNotFound     = sdkerrors.Register(ModuleName, 8, "account not found")
 	ErrSharesDecrease      = sdkerrors.Register(ModuleName, 9, "shares can't be decreased")
 	ErrVouchersMinting     = sdkerrors.Register(ModuleName, 10, "vouchers can't be minted")
+	ErrInvalidVouchers     = sdkerrors.Register(ModuleName, 11, "invalid vouchers")
+	ErrVouchersBurn        = sdkerrors.Register(ModuleName, 12, "vouchers can't be burned")
```

Comment:
Not needed, the bank's burn method should always succeed since funds are first sent to the module account. So handled with a critical error